docs/members-example/views/MembersCompositeView.tsx

```diff
@@ -17,7 +17,7 @@ export interface IMembersCompositeViewDispatchProps {
 
 export interface IMembersCompositeViewProps extends IMembersCompositeViewStateProps, IMembersCompositeViewDispatchProps, IReduxProps { }
 
-const mapStateToProps = (state: IReactVaporState): IMembersCompositeViewStateProps => {
+const mapStateToProps = (state: IReactVaporExampleState): IMembersCompositeViewStateProps => {
```

Comment:
We use certain reducers that are not part of the react-vapor component. They are only used in the examples. By using IReactVaporExampleState which extends from IReactVaporState, we do not pollute IReactVaporState with useless state attributes that won't be used when importing the interface.